### PR TITLE
Data Preparation documentation for AI chat [PAB-5234]

### PR DESCRIPTION
**Pull request by shaaf-c8y**

### CODE REVIEW

**Review by matj-sag — APPROVED** [review]

**Comment by matj-sag** [comment]
Have all the sprint review comments been completed? If so you can merge to the feature branch

File: content/data-preparation/rule-editor-bundle/ai-chat.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+### Using suggestions {#using-suggestions}
+
+After each response, the AI offers clickable **suggested next requests** that guide you toward common next steps. These suggestions are contextual — they change based on the current state of your code and conversation.
+
+Typical suggestions include:
+
+- Adding error handling for missing or null fields.
+- Adding an alarm for threshold conditions.
+- Adding a test for edge cases.
+- Generating test cases for the current code.
```
Comment (mcal-c8y) [inline content/data-preparation/rule-editor-bundle/ai-chat.md:83]: subjective but Im not sure a list of typical suggestions adds much value

File: content/data-preparation/rule-editor-bundle/ai-chat.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+Click any suggestion to send it as your next message. You can also ignore suggestions and type your own request.
```
Comment (mcal-c8y) [inline content/data-preparation/rule-editor-bundle/ai-chat.md:85]:
```suggestion
Click any suggestion to send it as your next message. You can also type your own request instead.
```
Something like this?
